src/components/AvatarImage.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang='ts'>
+  import type { IProfile } from "../db"
+  export let profile: IProfile;
+
+  $: robo = `https://robohash.org/${profile?.pubkey}.png`
```

Comment:
this queries the robohash regardless of it being needed or not. Please avoid that.